pkg/proxy/proxy.go

```diff
@@ -45,8 +45,8 @@ const (
 	fieldFd              = "fd"
 	fieldProxyRedirectID = "id"
 
-	// portReleaseDelay is the delay until a port is being released
-	portReleaseDelay = time.Duration(5) * time.Minute
+	// portReuseDelay is the delay until a port is being reused
+	portReuseDelay = time.Duration(5) * time.Minute
```

Comment:
You can remove the explicit cast to make this more readable:
```
portReuseDelay = 5 * time.Minute
```

Reply:
Thanks, and sorry for the hasted push.